crates/matrix-sdk-crypto/src/machine.rs

```diff
@@ -867,6 +867,7 @@ impl OlmMachine {
     }
 
     fn add_withheld_info(&self, changes: &mut Changes, event: &RoomKeyWithheldEvent) {
+        debug!(?event.content, "Processing `m.room_key.withheld` event");
```

Comment:
```suggestion
        debug!(?event.content, "Processing `m.room_key.withheld` event");
        
```

A bit of whitespace wouldn't hurt.